app/src/main/java/com/example/expenseappmvvm/data/database/entities/User.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.expenseappmvvm.data.database.entities
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "Users")
+data class User(
+    @PrimaryKey(autoGenerate = false) var uid: String = "",
```

Comment:
You'll not use Firebase so I guess this field can be autogenerated and ca be Long